### PR TITLE
coding with Python/using Mpris2

### PR DESCRIPTION
**Pull request by jamesaxl**

### CODE REVIEW

File: now-playing/frontend/command-manager/command-np/engine/mprisplayer.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#
+# 0. You just DO WHAT YOU WANT TO.
+# 1. Earth and all that is in it, with knowledge we protect it. And the best of mankind knowledge makes him and he makes it.
+
+import sys
+
+try:
+    import dbus
+except ImportError:
+    print (u"ProtocolManager.Command /me needs to install python-dbus :)")
```
Comment (meebey) [inline now-playing/frontend/command-manager/command-np/engine/mprisplayer.py]: change that to /echo else everyone will see the message

File: now-playing/frontend/command-manager/command-np/engine/mprisplayer.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    def __init__(self, mplayer):
+        session_bus = dbus.SessionBus()
+        player = session_bus.get_object('org.mpris.MediaPlayer2.%s' %mplayer, '/org/mpris/MediaPlayer2')
+        self.__iface = dbus.Interface(player, dbus_interface='org.freedesktop.DBus.Properties')
+
+    def GetTrackInfos(self):
+        try:
+            artist = self.__iface.Get('org.mpris.MediaPlayer2.Player','Metadata').get(dbus.String(u'xesam:artist'))[0]
+            if artist == None: artist = 'Unnown'
```
Comment (meebey) [inline now-playing/frontend/command-manager/command-np/engine/mprisplayer.py]: Typo: Unknown

File: now-playing/frontend/command-manager/command-np/engine/mprisplayer.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        except TypeError:
+            artist = 'Unnown'
+
+        try:
+            title = self.__iface.Get('org.mpris.MediaPlayer2.Player','Metadata').get(dbus.String(u'xesam:title'))
+            if title == None: title = 'Unnown'
```
Comment (meebey) [inline now-playing/frontend/command-manager/command-np/engine/mprisplayer.py]: Typo: Unknown

File: now-playing/frontend/command-manager/command-np/engine/__init__.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Everyone is permitted to copy and distribute verbatim or modified
+# copies of this license document, and changing it is allowed as long
+# as the name is changed.
+#
+# DO WHAT YOU WANT TO PUBLIC LICENSE
+# TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
+#
+# 0. You just DO WHAT YOU WANT TO.
+# 1. Earth and all that is in it, with knowledge we protect it. And the best of mankind knowledge makes him and he makes it.
```
Comment (meebey) [inline now-playing/frontend/command-manager/command-np/engine/__init__.py]: Don't change the license terms please, simply copy/paste it from the original now-playing hook

**Comment by jamesaxl** [comment]
change it

**Pull request merged**